### PR TITLE
Merge cv32e40s/release <-- cv32e40s/dev

### PR DESCRIPTION
**Pull request by MikeOpenHWGroup**
Step 1 in @silabs-robin's strategy to align the master branch with both cv32e40s/release and cv32e40s/dev _and_ restore Questa simulations.

### CODE REVIEW

**Comment by MikeOpenHWGroup** [comment]
Needed to fix a couple of merge conflicts, so I added this to a new branch, `cv32e40s/dev-1`.

File: bin/merge.sh [hunk 1]
```diff
@@ -19,18 +19,15 @@
 # limitations under the License.
```
Comment (silabs-robin) [inline bin/merge.sh:19]: Question:
Which parts of the "Files changed" are from `cv32e40s/dev` and which are from the manual conflict resolution?

Answer:
Diff of conflict resolution https://github.com/silabs-robin/core-v-verif/compare/release__merge_conflicts_from__dev..openhwgroup:core-v-verif:cv32e40s/dev-1

Comment (silabs-robin) [reply bin/merge.sh:19]: @MikeOpenHWGroup, this was the only manual changes you did, right?

Everything else in this PR's diff are changes that come from the `/dev` branch?

[the remark this answers is not in the document]
Comment (silabs-hfegran) [reply bin/merge.sh:19]: Sim appears to be good as well

**Comment by MikeOpenHWGroup** [comment]
Thanks for the quick review @silabs-robin and @silabs-hfegran.  Let me know if we can merge this. I'd like to move on to step 2...

File: bin/merge.sh (hunk 1 above)
Comment (silabs-robin) [reply bin/merge.sh:19]: And resolving the merge conflicts involves only the diff shown in the above link?

Comment (MikeOpenHWGroup) [reply bin/merge.sh:19]: Yes, that is correct.

**Review by silabs-robin — APPROVED** [review]
Approving based on:
Sim and formal is good.
Merge conflict is innocuous.
Rest of PR's diff is just new stuff from `/dev`.

**Pull request merged**